tools/go.mod

```diff
@@ -3,21 +3,13 @@ module github.com/hibiken/asynq/tools
 go 1.13
 
 require (
-	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
-	github.com/coreos/go-etcd v2.0.0+incompatible // indirect
-	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/fatih/color v1.9.0
-	github.com/go-redis/redis/v7 v7.4.0
-	github.com/golang/protobuf v1.4.1 // indirect
+	github.com/go-redis/redis/v8 v8.11.2
 	github.com/google/uuid v1.2.0
 	github.com/hibiken/asynq v0.17.1
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/spf13/cast v1.3.1
```

Comment:
I don't think this dependency should be removed. Would you mind running:

```sh
$ cd ./tools
$ go build ./...
```

Reply:
This completed successfully and did not result in any changes to go.mod or go.sum. I tried under both Go 1.15 and 1.16. Note that in Go 1.16 `go build` [no longer modifies](https://golang.org/doc/go1.16#go-command) go.mod or go.sum. 

> Build commands like go build and go test no longer modify go.mod and go.sum by default. Instead, they report an error if a module requirement or checksum needs to be added or updated (as if the -mod=readonly flag were used). Module requirements and sums may be adjusted with go mod tidy or go get.

Reply:
I see. I think I was wrong there. Please ignore this comment. But please fix the other comment and then we can merge this PR 👍